internal/app/machined/pkg/controllers/siderolink/manager.go

```diff
@@ -247,8 +341,51 @@ func (ctrl *ManagerController) Run(ctx context.Context, r controller.Runtime, lo
 			}); err != nil {
 			return fmt.Errorf("error creating address spec: %w", err)
 		}
```

Comment:
it might be nice to add some logging here for "success" case, e.g. `logger.Info("siderolink connection configured", zap.String("endpoint", endpoint), ...`

Reply:
done